README.rst

```diff
@@ -3,38 +3,21 @@ Armada
 
 |Docker Repository on Quay| |Build Status| |Doc Status|
 
-A python orchestrator for a installing, upgrading, and managing a
-collection of helm charts, dependencies, and values overrides.
-
-Note that this project is pre-alpha and under active development. It may
-undergo drastic changes to support the long-term vision but
-contributions are welcome.
-
-Overview
---------
-
-The armada python library and command line tool provides a way to
-synchronize a helm (tiller) target with an operators intended state,
-consisting of several charts, dependencies, and overrides using a single
-file or directory with a collection of files. This allows operators to
-define many charts, potentially with different namespaces for those
-releases, and their overrides in a central place. With a single command,
-deploy and/or upgrade them where applicable.
-
-Armada also supports fetching helm chart source and then building charts
-from source from various local and remote locations, such as git/github
-endpoints. In the future, it may supprot other mechanisms as well.
-
-It will also give the operator some indication of what is about to
-change by assisting with diffs for both values, values overrides, and
-actual template changes.
-
-Its functionality may extend beyond helm, assisting in interacting with
-kubernetes directly to perform basic pre and post steps, such as
-removing completed or failed jobs, running backup jobs, blocking on
-chart readiness, or deleting resources that do not support upgrades.
-However, primarily, it will be an interface to support orchestrating
-Helm.
+Armada is a tool for managing multiple helm charts with dependencies. It centralizes
+all chart configurations and versions in a single Armada file and provides pre/post
+hooks for installed components.
+
+Milestone
+---------
+
+Detailed Milestones can be viewed in:
+`Roadmap <https://github.com/att-comdev/armada/milestones>`_
+
+Issues can be filled in:
+`Issue <https://github.com/att-comdev/armada/issues>`_
+
+Getting Started
```

Comment:
Similar to the other projects, I'd like to see links here to additional docs in the tree instead of throwing everything in the README.  I feel the README should be a targeted tl;dr for the project with links to additional information, such as Getting Started, Installation, Troubleshooting, whatever else.